docs/colors.md

```diff
@@ -31,6 +31,18 @@ React Native supports also colors as an `int` values (in RGB color mode):
 
 > **_Note:_** This might appear similar to the Android [Color](https://developer.android.com/reference/android/graphics/Color) ints representation but on Android values are stored in SRGB color mode (0xaarrggbb).
 
+## DynamicColorIOS
```

Comment:
This could probably use some examples similar to what you have for PlatformColor

Reply:
I'll sync with tom to try and get an example piece in.